src/Game-core/sound.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "SFML/Graphics.hpp"
+#include "SFML/Audio.hpp"
+
+// In order for this script to work you will need to add additional dependencies 
+// properties --> linker --> input --> Additional dependencies --> sfml-audio-d.lib
+// whenever setting up project on different system remember to change sound paths
+#define eatSoundPath "C:/Users/HP/source/repos/sfml-snake/sounds/bite.wav"
+#define BGMPath "C:/Users/HP/source/repos/sfml-snake/sounds/BGM.wav"
+#define deathSoundPath "C:/Users/HP/source/repos/sfml-snake/sounds/death.wav"
+#define menuMusicPath "C:/Users/HP/source/repos/sfml-snake/sounds/mainMenu.wav"
+#define MAX_NUMBER_OF_ITEMS 3
+
+
+
+class sound
```

Comment:
`SoundController` or `SoundManager` will be the correct name. Since `Sound` hints to being an object wrapping over the `.wav` files.

Reply:
i changed the class name to **SoundController** , will it be okay if the file name remained the same.

Reply:
Ideally file name should reflect the class name. Since in this case only 1 class exists in the file I would strongly suggest renaming the file as well.